programs/uxd/tests/integration_tests/api/program_uxd/instructions/process_mint_with_credix_lp_depository.rs

```diff
@@ -142,13 +144,48 @@ pub async fn process_mint_with_credix_lp_depository(
             .await?
             .amount;
 
-    // Compute expected redeemable amount after minting fees
-    let redeemable_amount = calculate_amount_less_fees(
+    // Fetch information from onchain credix lp pool, to be able to predict precision loss
```

Comment:
This prooves that the precision loss maths testing is working now, because we need to take it into account, otherwise tests fails